src/surge-xt/gui/overlays/LuaEditors.cpp

```diff
@@ -791,15 +791,27 @@ void SurgeCodeEditorComponent::caretPositionMoved()
     }
 }
 
+void SurgeCodeEditorComponent::mouseWheelMove(const juce::MouseEvent &e,
+                                              const juce::MouseWheelDetails &wheel)
+{
+    auto modifiedMouseWheel = juce::MouseWheelDetails();
+    juce::MouseWheelDetails w(wheel);
+    if (e.mods.isAltDown())
```

Comment:
After trying this on Windows, no need to hide this behind Alt. Just multiply it 4x altogether.